docs/data/sdks/marketing-analytics-browser/index.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Marketing Analytics Browser
+description: The Amplitude Marketing Analytics Browser SDK Installation & Quick Start guide.
+icon: simple/typescript
+---
+
+
+![npm version](https://badge.fury.io/js/@amplitude%2Fmarketing-analytics-browser.svg)
+
+The Marketing Analytics Browser SDK extends the Browser SDK to identify users/events based on marketing channels. This library is open-source, check it out on [GitHub](https://github.com/amplitude/Amplitude-TypeScript/tree/main/packages/marketing-analytics-browser).
+
+!!!info "Marketing Analytics Browser SDK Resources"
+    [:material-github: Github](https://github.com/amplitude/Amplitude-TypeScript/tree/main/packages/marketing-analytics-browser) · [:material-code-tags-check: Releases](https://github.com/amplitude/Amplitude-TypeScript/releases?q=marketing-analytics-browser&expanded=true) · [:material-book: API Reference](https://amplitude.github.io/Amplitude-TypeScript/modules/_amplitude_marketing_analytics_browser.html)
+
+!!!note "Marketing Analytics Browser versus the Typescript Broswer SDK"
+    Marketing Analytics Browser SDK extend the Typescript Broswer SDK but with tracking web attribution and tracking page view support.
```

Comment:
I agree with @caseyamp I think this section could be updated to include:
1. When should customers use this vs the standard SDK. For fast automatic tracking?
2. A note about this SDK automatically including marketing plugins
3. Link to the standard SDK
4. Link to Plugins

Reply:
Not talking about plugins here, because we thought if people knew those are plugins, they might install the typescript browser and the two plugins separately. Then we are not able to track the marketing analytics traffic. Because people only install the marketing analytics package will have the `amplitude-ma-ts` context, which we are using for the matrix. Let me know what you think. @justin-fiedler 
cc:  @kevinpagtakhan

Reply:
My impression was the ma sdk was for low-tech customers that want an all in one tracking solution similar to google analytics. If the customer wants to deal with plugins, it's the better long term solution. Also the MA SDK isn't compatible with the ampli cli

Reply:
> Not talking about plugins here, because we thought if people knew those are plugins, they might install the typescript browser and the two plugins separately. Then we are not able to track the marketing analytics traffic. Because people only install the marketing analytics package will have the amplitude-ma-ts context, which we are using for the matrix. Let me know what you think.

Got it. Agree it makes sense to not mention plugins here as not to confuse non-technical users. I provided a suggested update to the note in TS SDK.